### ISSUE TITLE
Error with PyInstaller

### ISSUE THREAD

**Issue by therealbaljeet** [codelucas/newspaper#877]
After converting a (working) simple demo to an exe using the source code in the README, I ran into the following error when running the binary:

```FileNotFoundError: [WinError 3] The system cannot find the path specified 'C:\\Users\\****\\AppData\\Local\\Temp\\_MEI212082\newspaper\resources/text```

What would I have to manipulate in the source code to fix this? I saw a similar issue somewhere here (I'll link to it if I can find it again), but there was no solution given.

**Comment by lalo-AI** [comment]
HI, 

I have the same issue, until now I haven't been able to solve this. I hope someone can help us.

**Comment by http-samc** [comment]
Yup. Really want to use this in a desktpp app, but right now I have to resort to putting it on a server and making a flask API just to provide the functionality, which is coming out of my pocket.
